django/contrib/admin/options.py

```diff
@@ -1038,11 +1054,25 @@ def get_actions(self, request):
         if self.actions is None or IS_POPUP_VAR in request.GET:
             return {}
         actions = self._filter_actions_by_permissions(request, self._get_base_actions())
+
+        return {name: (func, name, desc) for func, name, desc in actions}
+
+    def get_detail_actions(self, request):
+        """
+        Same as `get_actions` but for `detail_actions`.
```

Comment:
Not sure if it is a good docstring. The only difference with `get_actions` is that it uses `_get_detail_actions`, but we need another method in case user wants overwrite it